### DISCUSSION TITLE
Ikea Tretakt E2204 not working after pairing

### DISCUSSION THREAD

**Discussion by befedi** [Koenkk/zigbee2mqtt#24983] [Devices]
Hello together,

I am having problems integrating an Ikea Tretakt.

It seems to configure correctly but I am not able to switch it via Frontend. I always get an error message for the get and set reguests like:

`Publish 'set' 'state' to '0x881a14affe2f369c' failed: 'Error: ZCL command 0x881a14fffe2f479c/1 genOnOff.on({}, {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (Timeout - 4174 - 1 - 232 - 6 - 11 after 10000ms)' `

I also have tried to re-pair it multiple times but with the same result.

Does anybody has an idea?

**Comment by Tobi-St** [comment]
Hey, same here! It actually works for just a few seconds after pairing, but then I get the same error. Did you maybe find a way to solve it?

**Reply by befedi** [reply]
Hi, I have not found a solution, yet.

**Comment by befedi** [comment]
I now have updated my setup to:
- z2m: 1.42.0 (commit: 861cba6)
- SONOFF ZBDongle-P: 20240710

Everything (all other devices) are working fine but now I get the following issue while pairing the E2204:
`zh:controller: Interview failed for '0x881a14fffe2f6181 with error 'Error: AREQ - ZDO - simpleDescRsp after 10000ms'`
`z2m: Failed to interview '0x881a14fffe2f6181', device has not successfully been paired`

**Comment by jeroenbeuz** [comment]
I was having the same issues. The device would pair just fine but after a couple of seconds it wouldn't response any more. The fix for me was to first hit the On Off toggle a couple of times **before** you rename the device in Z2M. The device should respond. Then rename the device and it should still work. I've tried this with 2 Tretakt plugs and can reproduce on both. Hope it also works for you.

#Edit: tried it another couple of times and apparently my method is not working. It's a hit or miss, just repair when it stops working until it does seems to be the only workaround.

**Comment by lukasdietz** [comment]
Is there an update to this issue?
I have the same problem. Right after pairing the switch is working for around 30 seconds and suddenly stops responding with timeout errors. As this happened to me with two devices I think it is no device failure but rather a bug.

I am using ZBT-2 with ember.

**Reply by andrei-lazarov** [reply]
The switch? You mean the plug?
Sounds like a Network error. Coordinator can't reach device. 
Maybe it's going through a bad router

**Reply by lukasdietz** [reply]
Yes I mean the plug.
Could be but I honestly can't imagine because it works right after pairing and always stops after a short time. The plug is directly paired to the zigbee antenna.

Would it be helpful to sniff the zigbee traffic?

**Reply by andrei-lazarov** [reply]
Yes definitely. My E2204 plugs work well
